docs/aws/setup-testnet-validator-from-scratch.md

```diff
@@ -415,6 +267,37 @@ You should see your IP address on the list
 curl -s http://127.0.0.1:8888/status
 ```
 
+#### Monitor the node's sync progres
+You can monitor the node's synchronization progress by using the ```node_util.py``` utility script again:
+
+```
+/etc/casper/node_util.py watch
+```
+
+When you run the watch command, expect to see something like this:
+```
+Last Block: 630151 (Era: 4153)
+Peer Count: 297
+Uptime: 4days 6h 40m 18s 553ms
+Build: 1.4.5-a7f6a648d-casper-mainnet
+Key: 0147b4cae09d64ab6acd02dd0868722be9a9bcc355c2fdff7c2c244cbfcd30f158
+Next Upgrade: None
+
+RPC: Ready
+
+● casper-node-launcher.service - Casper Node Launcher
+   Loaded: loaded (/lib/systemd/system/casper-node-launcher.service; enabled; vendor preset: enabled)
+   Active: active (running) since Wed 2022-03-16 21:08:50 UTC; 4 days ago
+     Docs: https://docs.casperlabs.io
+ Main PID: 2934 (casper-node-lau)
+    Tasks: 12 (limit: 4915)
+   CGroup: /system.slice/casper-node-launcher.service
+           ├─ 2934 /usr/bin/casper-node-launcher
+           └─16842 /var/lib/casper/bin/1_4_5/casper-node validator /etc/casper/1_4_5/config.toml
+```
+
+If your casper-node-launcher status is not active (running) with increasing time, you are either not running or restarting.
+
```

Comment:
```suggestion

The watch command also allows an `--ip` argument to use with a node on the same network that is in sync.  This will show how far behind your node currently is.
```